lib/components/MenuItemDivider/MenuItemDivider.tsx

```diff
@@ -0,0 +1,17 @@
+import assert from 'assert';
+import React, { useContext } from 'react';
+import { Box, Divider } from '..';
+import { MenuRendererContext } from '../MenuRenderer/MenuRenderer';
+
+export const MenuItemDivider = () => {
+  assert(
+    useContext(MenuRendererContext),
+    'MenuItemDivider must be rendered within a menu component. See the documentation for correct usage: https://seek-oss.github.io/braid-design-system/components/MenuItemDivider',
+  );
+
+  return (
+    <Box padding="xxsmall">
```

Comment:
Feel like this padding should be explicitly shared with the [vertical gutter](https://github.com/seek-oss/braid-design-system/pull/751/files#diff-94ff30bdbaca5a1466f37b3964103c89R298) on the list